.babelrc

```diff
@@ -1,11 +1,26 @@
 {
-  "presets": [
-    "es2015"
-  ],
   "env": {
```

Comment:
Just a comment and an improvement in the future we should reduce the amount of `envs` here. when we review the mocking in the unit test